### ISSUE TITLE
Profile Team | Sprint 85 | Oct 19 - Nov 1

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#47872]
## PTO/OOO/Unavailable

- **Adam**: 
- **Christina**:  
- **Sharon**:  
- **Tom**: 
- **Samara**: 10/19 - 10/21 (OCTO "offsite" -  limited availability)
- **Liz**:  10/28 - out of office majority of the day
- **John**:

## Retro Board is open!:
- [ ] [Sprint 84](https://www.reetro.app/board/6203c8f3edfa6100174612bb/633c6fdadb1fde17f3c8b321)

---

## Sprint Priorities:
- Continue roll-out of BAI
- Accessibility audit work
- Research recommendations
---

#### FE - Adam
- [x] #44759 release to 100%
- [x] #48273 BAI documentation (1 pt)

#### PM - Sharon
- [x] #48272 - update PO for Contact info and BAI project

#### Design - Christina
- [x] #48271 BAI Documentation (2 pts)

#### PM - Sharon
- [x] #47649 Documentation update (audit) 

#### BE - Tom
- [x] #48365 BE Document for Account Security (2 pts)

#### FE - Adam 
- [x] #48381 Connected Apps (1 pt)

#### Design - Liz
- [x] #48396 Notifications Settings Language (3 pts) 

#### Design - Christina
- [x] #47266 Nametag discovery work - landmark role (defect-3) (5 pts)

#### FE -  John
- [x] #37363 Connected apps (2 pts)

#### Design -  Christina
- [x] #48151 Fiduciary flag UI for all of Profile (tbd)
- [x] #48264 Nametag | Discovery (4 pts) (possible carry-over to Sprint 86)
- [x] #47350 Update MPI error alerts (2 pts)
- [x] #48460 Update CC guide (2 pts)

#### BE - Tom
- [x] #43910 Notification Preferences - Add Default Send Indicator (Opt-in) (3 pts)
- [x] #37483 Confirmation Emails | Include Which # and//or address was updated (2 pts)
- [x] #48225 Military Improvements to Service History records (3 pts)

#### FE  - Adam 
- [x] #48202 Feature Flag for new BVA Notification (1 pt))
- [x] #48147 Create custom event to track submits + # of in line errors (2 pt)
- [x] #44886 Update UI Alerts for Address Validation Use Cases (3 pts)

#### FE - John
- [ ] #47477 Military/Alert updates (2 pts) - pending text tweaks (moved to sprint 86)

#### Design - Liz
~- [ ] #48274 Prepare research presentation artifacts (3 pts)~ (moving out to later sprint)

#### PM - Erica
- [x] #48437 Explore Analytics Around Adding Mobile Number Flow

#### PM - Sharon
- [x] #48439  Review Analytics For Editing Addresses

#### Design - Christina
- [x] #46564 Update FAQ for DD214

**Comment by Samara-Strauss** [comment]
[Cross-posting from Slack so I don't forget](https://dsva.slack.com/archives/C909ZG2BB/p1665683110861219): we need a ticket for next sprint to add a feature flag for the new BVA notification so it doesn't surprisingly show up in any of our environments before we're ready while VA Profile is doing testing and making adjustments. [As a refresher, this is the notification I am talking about](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47441). Let me know if you have questions!
